src/nvme/tree.c

```diff
@@ -1784,6 +1784,10 @@ static char *nvme_ctrl_lookup_phy_slot(nvme_root_t r, const char *address)
 				return NULL;
 			}
 			addr = nvme_get_attr(path, "address");
+
+			/* some directories don't have an address entry */
+			if (addr == NULL)
```

Comment:
A cleaner way to do this would be ```if (!addr)```

Reply:
I guess that I'm a K&R guy but if that's the style for the library, I can change it.